### PR TITLE
testing/fbgrab: new aport

### PR DESCRIPTION
**Pull request by durrendal**
New APKBUILD for Gunnar Monell's fbgrab

### CODE REVIEW

File: testing/fbgrab/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+pkgname=fbgrab
+pkgver=1.3
+pkgrel=0
+pkgdesc="Framebuffer screenshot program"
+url="https://fbgrab.monells.se"
+arch="all"
+license="GPL-2.0-only"
+depends="zlib libpng libpng-dev"
```
Comment (maxice8) [inline testing/fbgrab/APKBUILD]: that looks definitively  wrong.

`zlib-dev` and `libpng-dev` should be in `makedepends`

Comment (maxice8) [reply testing/fbgrab/APKBUILD]: Also no  `zlib` and `libpng`  in `depends`

File: testing/fbgrab/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+pkgname=fbgrab
+pkgver=1.3
+pkgrel=0
+pkgdesc="Framebuffer screenshot program"
+url="https://fbgrab.monells.se"
+arch="all"
+license="GPL-2.0-only"
+depends=""
+makedepends="zlib zlib-dev libpng libpng-dev gcc build-base"
```
Comment (maxice8) [inline testing/fbgrab/APKBUILD]: should be

zlib-dev libpng-dev only

**Comment by maxice8** [comment]
You're missing `gzip` and `linux-headers` on `makedepends`

Please add a -doc subpackage

**Comment by durrendal** [comment]
I appreciate your help Maxice, I'll be sure to clean up the mess I made once the build process passes, and everything looks sane.

**Comment by durrendal** [comment]
Speaking on build processes,

It looks like the ones going on now are throwing back this error


### Build summary ###
>>> No packages found to be built.

File: testing/fbgrab/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+pkgname=fbgrab
+pkgver=1.3
+pkgrel=0
+pkgdesc="Framebuffer screenshot program"
+url="https://fbgrab.monells.se"
+arch="all"
+license="GPL-2.0-only"
+depends=""
+makedepends="zlib-dev libpng-dev gzip linux-headers"
+subpackages=$pkgname-doc"
```
Comment (maxice8) [inline testing/fbgrab/APKBUILD]: missing `"`

File: testing/fbgrab/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+pkgname=fbgrab
+pkgver=1.3
+pkgrel=0
+pkgdesc="Framebuffer screenshot program"
+url="https://fbgrab.monells.se"
+arch="all"
+license="GPL-2.0-only"
+depends=""
```
Comment (maxice8) [inline testing/fbgrab/APKBUILD]: empty variable can be removed

**Comment by durrendal** [comment]
Okay, empty depends is gone, and the missing " is now back. Hopefully that's all that's needed and I can get this squashed

**Pull request closed without merging**